Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 5
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	99	discretize_counts
	100

[Fri Jan 29 23:22:43 2021]
rule discretize_counts:
    input: read_counts/AML-28-001_AD.csv, read_counts/AML-28-001_DP.csv
    output: input/AML-28-001_input.csv, input/AML-28-001.loci
    jobid: 30
    wildcards: patient=AML-28-001

[Fri Jan 29 23:22:43 2021]
rule discretize_counts:
    input: read_counts/AML-76-001_AD.csv, read_counts/AML-76-001_DP.csv
    output: input/AML-76-001_input.csv, input/AML-76-001.loci
    jobid: 76
    wildcards: patient=AML-76-001

[Fri Jan 29 23:22:43 2021]
rule discretize_counts:
    input: read_counts/AML-26-001_AD.csv, read_counts/AML-26-001_DP.csv
    output: input/AML-26-001_input.csv, input/AML-26-001.loci
    jobid: 28
    wildcards: patient=AML-26-001

[Fri Jan 29 23:22:43 2021]
rule discretize_counts:
    input: read_counts/AML-01-001_AD.csv, read_counts/AML-01-001_DP.csv
    output: input/AML-01-001_input.csv, input/AML-01-001.loci
    jobid: 1
    wildcards: patient=AML-01-001


[Fri Jan 29 23:22:43 2021]
rule discretize_counts:
    input: read_counts/AML-06-001_AD.csv, read_counts/AML-06-001_DP.csv
    output: input/AML-06-001_input.csv, input/AML-06-001.loci
    jobid: 5
    wildcards: patient=AML-06-001

[Fri Jan 29 23:22:48 2021]
Error in rule discretize_counts:
    jobid: 1
    output: input/AML-01-001_input.csv, input/AML-01-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpkmnzchun.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:48 2021]
rule discretize_counts:
    input: read_counts/AML-31-001_AD.csv, read_counts/AML-31-001_DP.csv
    output: input/AML-31-001_input.csv, input/AML-31-001.loci
    jobid: 33
    wildcards: patient=AML-31-001

[Fri Jan 29 23:22:48 2021]
Error in rule discretize_counts:
    jobid: 5
    output: input/AML-06-001_input.csv, input/AML-06-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpxfycraoc.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:48 2021]
rule discretize_counts:
    input: read_counts/AML-66-001_AD.csv, read_counts/AML-66-001_DP.csv
    output: input/AML-66-001_input.csv, input/AML-66-001.loci
    jobid: 66
    wildcards: patient=AML-66-001

[Fri Jan 29 23:22:48 2021]
Error in rule discretize_counts:
    jobid: 30
    output: input/AML-28-001_input.csv, input/AML-28-001.loci

[Fri Jan 29 23:22:48 2021]
RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp_qjy87bu.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Error in rule discretize_counts:
    jobid: 28
    output: input/AML-26-001_input.csv, input/AML-26-001.loci

Job failed, going on with independent jobs.

[Fri Jan 29 23:22:48 2021]
rule discretize_counts:
    input: read_counts/AML-77-001_AD.csv, read_counts/AML-77-001_DP.csv
    output: input/AML-77-001_input.csv, input/AML-77-001.loci
    jobid: 77
    wildcards: patient=AML-77-001
RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7l2p5q_h.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run

Job failed, going on with independent jobs.

[Fri Jan 29 23:22:48 2021]
rule discretize_counts:
    input: read_counts/AML-51-001_AD.csv, read_counts/AML-51-001_DP.csv
    output: input/AML-51-001_input.csv, input/AML-51-001.loci
    jobid: 51
    wildcards: patient=AML-51-001

[Fri Jan 29 23:22:49 2021]
Error in rule discretize_counts:
    jobid: 76
    output: input/AML-76-001_input.csv, input/AML-76-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpk6cs7p0j.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:49 2021]
rule discretize_counts:
    input: read_counts/AML-74-001_AD.csv, read_counts/AML-74-001_DP.csv
    output: input/AML-74-001_input.csv, input/AML-74-001.loci
    jobid: 74
    wildcards: patient=AML-74-001

[Fri Jan 29 23:22:50 2021]
Error in rule discretize_counts:
    jobid: 51
    output: input/AML-51-001_input.csv, input/AML-51-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp2lsllafm.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:50 2021]
rule discretize_counts:
    input: read_counts/AML-43-001_AD.csv, read_counts/AML-43-001_DP.csv
    output: input/AML-43-001_input.csv, input/AML-43-001.loci
    jobid: 44
    wildcards: patient=AML-43-001

[Fri Jan 29 23:22:53 2021]
Error in rule discretize_counts:
    jobid: 33
    output: input/AML-31-001_input.csv, input/AML-31-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp6vy4apop.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:53 2021]
rule discretize_counts:
    input: read_counts/AML-14-001_AD.csv, read_counts/AML-14-001_DP.csv
    output: input/AML-14-001_input.csv, input/AML-14-001.loci
    jobid: 16
    wildcards: patient=AML-14-001

[Fri Jan 29 23:22:53 2021]
Error in rule discretize_counts:
    jobid: 77
    output: input/AML-77-001_input.csv, input/AML-77-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpe8j0nku0.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:53 2021]
rule discretize_counts:
    input: read_counts/AML-73-001_AD.csv, read_counts/AML-73-001_DP.csv
    output: input/AML-73-001_input.csv, input/AML-73-001.loci
    jobid: 73
    wildcards: patient=AML-73-001

[Fri Jan 29 23:22:53 2021]
Error in rule discretize_counts:
    jobid: 74
    output: input/AML-74-001_input.csv, input/AML-74-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpvi20qdun.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:53 2021]
rule discretize_counts:
    input: read_counts/AML-32-001_AD.csv, read_counts/AML-32-001_DP.csv
    output: input/AML-32-001_input.csv, input/AML-32-001.loci
    jobid: 34
    wildcards: patient=AML-32-001

[Fri Jan 29 23:22:54 2021]
Error in rule discretize_counts:
    jobid: 66
    output: input/AML-66-001_input.csv, input/AML-66-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp4hzar8li.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:54 2021]
rule discretize_counts:
    input: read_counts/AML-90-001_AD.csv, read_counts/AML-90-001_DP.csv
    output: input/AML-90-001_input.csv, input/AML-90-001.loci
    jobid: 90
    wildcards: patient=AML-90-001

[Fri Jan 29 23:22:54 2021]
Error in rule discretize_counts:
    jobid: 44
    output: input/AML-43-001_input.csv, input/AML-43-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpzppd7aps.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:54 2021]
rule discretize_counts:
    input: read_counts/AML-41-001_AD.csv, read_counts/AML-41-001_DP.csv
    output: input/AML-41-001_input.csv, input/AML-41-001.loci
    jobid: 42
    wildcards: patient=AML-41-001

[Fri Jan 29 23:22:57 2021]
Error in rule discretize_counts:
    jobid: 73
    output: input/AML-73-001_input.csv, input/AML-73-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpn_n52ra5.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:57 2021]
rule discretize_counts:
    input: read_counts/AML-09-001_AD.csv, read_counts/AML-09-001_DP.csv
    output: input/AML-09-001_input.csv, input/AML-09-001.loci
    jobid: 8
    wildcards: patient=AML-09-001

[Fri Jan 29 23:22:58 2021]
Error in rule discretize_counts:
    jobid: 16
    output: input/AML-14-001_input.csv, input/AML-14-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp3t4c3s9v.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:58 2021]
rule discretize_counts:
    input: read_counts/AML-67-001_AD.csv, read_counts/AML-67-001_DP.csv
    output: input/AML-67-001_input.csv, input/AML-67-001.loci
    jobid: 67
    wildcards: patient=AML-67-001

[Fri Jan 29 23:22:58 2021]
Error in rule discretize_counts:
    jobid: 34
    output: input/AML-32-001_input.csv, input/AML-32-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpw76u9sz3.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:58 2021]
rule discretize_counts:
    input: read_counts/AML-02-001_AD.csv, read_counts/AML-02-001_DP.csv
    output: input/AML-02-001_input.csv, input/AML-02-001.loci
    jobid: 2
    wildcards: patient=AML-02-001

[Fri Jan 29 23:22:58 2021]
Error in rule discretize_counts:
    jobid: 90
    output: input/AML-90-001_input.csv, input/AML-90-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpa08ektxz.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:58 2021]
rule discretize_counts:
    input: read_counts/AML-83-001_AD.csv, read_counts/AML-83-001_DP.csv
    output: input/AML-83-001_input.csv, input/AML-83-001.loci
    jobid: 83
    wildcards: patient=AML-83-001

[Fri Jan 29 23:22:59 2021]
Error in rule discretize_counts:
    jobid: 42
    output: input/AML-41-001_input.csv, input/AML-41-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpb7ybn8bj.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:22:59 2021]
rule discretize_counts:
    input: read_counts/AML-13-001_AD.csv, read_counts/AML-13-001_DP.csv
    output: input/AML-13-001_input.csv, input/AML-13-001.loci
    jobid: 15
    wildcards: patient=AML-13-001

[Fri Jan 29 23:23:02 2021]
Error in rule discretize_counts:
    jobid: 83
    output: input/AML-83-001_input.csv, input/AML-83-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpyl1z_ovp.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:02 2021]
rule discretize_counts:
    input: read_counts/AML-40-001_AD.csv, read_counts/AML-40-001_DP.csv
    output: input/AML-40-001_input.csv, input/AML-40-001.loci
    jobid: 41
    wildcards: patient=AML-40-001

[Fri Jan 29 23:23:02 2021]
Error in rule discretize_counts:
    jobid: 8
    output: input/AML-09-001_input.csv, input/AML-09-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmprwg7_97f.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:02 2021]
rule discretize_counts:
    input: read_counts/AML-100-001_AD.csv, read_counts/AML-100-001_DP.csv
    output: input/AML-100-001_input.csv, input/AML-100-001.loci
    jobid: 9
    wildcards: patient=AML-100-001

[Fri Jan 29 23:23:02 2021]
Error in rule discretize_counts:
    jobid: 2
    output: input/AML-02-001_input.csv, input/AML-02-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpoorolqaz.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:02 2021]
rule discretize_counts:
    input: read_counts/AML-39-001_AD.csv, read_counts/AML-39-001_DP.csv
    output: input/AML-39-001_input.csv, input/AML-39-001.loci
    jobid: 40
    wildcards: patient=AML-39-001

[Fri Jan 29 23:23:02 2021]
Error in rule discretize_counts:
    jobid: 67
    output: input/AML-67-001_input.csv, input/AML-67-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmps27al_sj.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:02 2021]
rule discretize_counts:
    input: read_counts/AML-85-001_AD.csv, read_counts/AML-85-001_DP.csv
    output: input/AML-85-001_input.csv, input/AML-85-001.loci
    jobid: 85
    wildcards: patient=AML-85-001

[Fri Jan 29 23:23:03 2021]
Error in rule discretize_counts:
    jobid: 15
    output: input/AML-13-001_input.csv, input/AML-13-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7r5eod6g.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:03 2021]
rule discretize_counts:
    input: read_counts/AML-98-001_AD.csv, read_counts/AML-98-001_DP.csv
    output: input/AML-98-001_input.csv, input/AML-98-001.loci
    jobid: 98
    wildcards: patient=AML-98-001

[Fri Jan 29 23:23:06 2021]
Error in rule discretize_counts:
    jobid: 85
    output: input/AML-85-001_input.csv, input/AML-85-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp5tm1qdbm.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:06 2021]
rule discretize_counts:
    input: read_counts/AML-63-001_AD.csv, read_counts/AML-63-001_DP.csv
    output: input/AML-63-001_input.csv, input/AML-63-001.loci
    jobid: 63
    wildcards: patient=AML-63-001

[Fri Jan 29 23:23:06 2021]
Error in rule discretize_counts:
    jobid: 9
    output: input/AML-100-001_input.csv, input/AML-100-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpwnpj4si4.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:06 2021]
rule discretize_counts:
    input: read_counts/AML-82-001_AD.csv, read_counts/AML-82-001_DP.csv
    output: input/AML-82-001_input.csv, input/AML-82-001.loci
    jobid: 82
    wildcards: patient=AML-82-001

[Fri Jan 29 23:23:06 2021]
Error in rule discretize_counts:
    jobid: 41
    output: input/AML-40-001_input.csv, input/AML-40-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7vu0n_bh.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:06 2021]
rule discretize_counts:
    input: read_counts/AML-70-001_AD.csv, read_counts/AML-70-001_DP.csv
    output: input/AML-70-001_input.csv, input/AML-70-001.loci
    jobid: 70
    wildcards: patient=AML-70-001

[Fri Jan 29 23:23:07 2021]
Error in rule discretize_counts:
    jobid: 40
    output: input/AML-39-001_input.csv, input/AML-39-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpzwu6t3d5.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:07 2021]
rule discretize_counts:
    input: read_counts/AML-92-001_AD.csv, read_counts/AML-92-001_DP.csv
    output: input/AML-92-001_input.csv, input/AML-92-001.loci
    jobid: 92
    wildcards: patient=AML-92-001

[Fri Jan 29 23:23:07 2021]
Error in rule discretize_counts:
    jobid: 98
    output: input/AML-98-001_input.csv, input/AML-98-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmprk8hmnxv.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:07 2021]
rule discretize_counts:
    input: read_counts/AML-16-001_AD.csv, read_counts/AML-16-001_DP.csv
    output: input/AML-16-001_input.csv, input/AML-16-001.loci
    jobid: 18
    wildcards: patient=AML-16-001

[Fri Jan 29 23:23:10 2021]
Error in rule discretize_counts:
    jobid: 82
    output: input/AML-82-001_input.csv, input/AML-82-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpgsj93zgo.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:10 2021]
[Fri Jan 29 23:23:10 2021]
rule discretize_counts:
    input: read_counts/AML-20-001_AD.csv, read_counts/AML-20-001_DP.csv
    output: input/AML-20-001_input.csv, input/AML-20-001.loci
    jobid: 22
    wildcards: patient=AML-20-001
Error in rule discretize_counts:

    jobid: 63
    output: input/AML-63-001_input.csv, input/AML-63-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpx39u35go.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:10 2021]
rule discretize_counts:
    input: read_counts/AML-88-001_AD.csv, read_counts/AML-88-001_DP.csv
    output: input/AML-88-001_input.csv, input/AML-88-001.loci
    jobid: 88
    wildcards: patient=AML-88-001

[Fri Jan 29 23:23:10 2021]
Error in rule discretize_counts:
    jobid: 70
    output: input/AML-70-001_input.csv, input/AML-70-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpyqlqtcvh.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:10 2021]
[Fri Jan 29 23:23:10 2021]
rule discretize_counts:
    input: read_counts/AML-50-001_AD.csv, read_counts/AML-50-001_DP.csv
    output: input/AML-50-001_input.csv, input/AML-50-001.loci
    jobid: 50
    wildcards: patient=AML-50-001
Error in rule discretize_counts:

    jobid: 18
    output: input/AML-16-001_input.csv, input/AML-16-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpw5pue9o5.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:10 2021]
rule discretize_counts:
    input: read_counts/AML-45-001_AD.csv, read_counts/AML-45-001_DP.csv
    output: input/AML-45-001_input.csv, input/AML-45-001.loci
    jobid: 45
    wildcards: patient=AML-45-001

[Fri Jan 29 23:23:11 2021]
Error in rule discretize_counts:
    jobid: 92
    output: input/AML-92-001_input.csv, input/AML-92-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp_mpxat96.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:11 2021]
rule discretize_counts:
    input: read_counts/AML-96-001_AD.csv, read_counts/AML-96-001_DP.csv
    output: input/AML-96-001_input.csv, input/AML-96-001.loci
    jobid: 96
    wildcards: patient=AML-96-001

[Fri Jan 29 23:23:14 2021]
Error in rule discretize_counts:
    jobid: 50
    output: input/AML-50-001_input.csv, input/AML-50-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpyf6i1tru.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:14 2021]
rule discretize_counts:
    input: read_counts/AML-60-001_AD.csv, read_counts/AML-60-001_DP.csv
    output: input/AML-60-001_input.csv, input/AML-60-001.loci
    jobid: 60
    wildcards: patient=AML-60-001

[Fri Jan 29 23:23:14 2021]
Error in rule discretize_counts:
    jobid: 22
    output: input/AML-20-001_input.csv, input/AML-20-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp69f0m64h.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:14 2021]
rule discretize_counts:
    input: read_counts/AML-95-001_AD.csv, read_counts/AML-95-001_DP.csv
    output: input/AML-95-001_input.csv, input/AML-95-001.loci
    jobid: 95
    wildcards: patient=AML-95-001

[Fri Jan 29 23:23:14 2021]
Error in rule discretize_counts:
    jobid: 45
    output: input/AML-45-001_input.csv, input/AML-45-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpemv5x2h3.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:14 2021]
rule discretize_counts:
    input: read_counts/AML-29-001_AD.csv, read_counts/AML-29-001_DP.csv
    output: input/AML-29-001_input.csv, input/AML-29-001.loci
    jobid: 31
    wildcards: patient=AML-29-001

[Fri Jan 29 23:23:14 2021]
Error in rule discretize_counts:
    jobid: 88
    output: input/AML-88-001_input.csv, input/AML-88-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpsaz_38_z.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:14 2021]
rule discretize_counts:
    input: read_counts/AML-69-001_AD.csv, read_counts/AML-69-001_DP.csv
    output: input/AML-69-001_input.csv, input/AML-69-001.loci
    jobid: 69
    wildcards: patient=AML-69-001

[Fri Jan 29 23:23:15 2021]
Error in rule discretize_counts:
    jobid: 96
    output: input/AML-96-001_input.csv, input/AML-96-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpwdy_sj26.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:15 2021]
rule discretize_counts:
    input: read_counts/AML-21-001_AD.csv, read_counts/AML-21-001_DP.csv
    output: input/AML-21-001_input.csv, input/AML-21-001.loci
    jobid: 23
    wildcards: patient=AML-21-001

[Fri Jan 29 23:23:16 2021]
Error in rule discretize_counts:
    jobid: 60
    output: input/AML-60-001_input.csv, input/AML-60-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmph6pf5ddr.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:16 2021]
rule discretize_counts:
    input: read_counts/AML-57-001_AD.csv, read_counts/AML-57-001_DP.csv
    output: input/AML-57-001_input.csv, input/AML-57-001.loci
    jobid: 57
    wildcards: patient=AML-57-001

[Fri Jan 29 23:23:17 2021]
Error in rule discretize_counts:
    jobid: 95
    output: input/AML-95-001_input.csv, input/AML-95-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpagenzz4b.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:17 2021]
rule discretize_counts:
    input: read_counts/AML-99-001_AD.csv, read_counts/AML-99-001_DP.csv
    output: input/AML-99-001_input.csv, input/AML-99-001.loci
    jobid: 99
    wildcards: patient=AML-99-001

[Fri Jan 29 23:23:18 2021]
Error in rule discretize_counts:
    jobid: 31
    output: input/AML-29-001_input.csv, input/AML-29-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmphk9it2f0.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:18 2021]
rule discretize_counts:
    input: read_counts/AML-94-001_AD.csv, read_counts/AML-94-001_DP.csv
    output: input/AML-94-001_input.csv, input/AML-94-001.loci
    jobid: 94
    wildcards: patient=AML-94-001

[Fri Jan 29 23:23:18 2021]
Error in rule discretize_counts:
    jobid: 69
    output: input/AML-69-001_input.csv, input/AML-69-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpkragxrem.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:18 2021]
rule discretize_counts:
    input: read_counts/AML-36-001_AD.csv, read_counts/AML-36-001_DP.csv
    output: input/AML-36-001_input.csv, input/AML-36-001.loci
    jobid: 38
    wildcards: patient=AML-36-001

[Fri Jan 29 23:23:18 2021]
Error in rule discretize_counts:
    jobid: 23
    output: input/AML-21-001_input.csv, input/AML-21-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7y7lul4r.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:18 2021]
rule discretize_counts:
    input: read_counts/AML-49-001_AD.csv, read_counts/AML-49-001_DP.csv
    output: input/AML-49-001_input.csv, input/AML-49-001.loci
    jobid: 49
    wildcards: patient=AML-49-001

[Fri Jan 29 23:23:19 2021]
Error in rule discretize_counts:
    jobid: 57
    output: input/AML-57-001_input.csv, input/AML-57-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7vj5aca7.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:19 2021]
rule discretize_counts:
    input: read_counts/AML-58-001_AD.csv, read_counts/AML-58-001_DP.csv
    output: input/AML-58-001_input.csv, input/AML-58-001.loci
    jobid: 58
    wildcards: patient=AML-58-001

[Fri Jan 29 23:23:20 2021]
Error in rule discretize_counts:
    jobid: 49
    output: input/AML-49-001_input.csv, input/AML-49-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpxteu4tg8.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:20 2021]
rule discretize_counts:
    input: read_counts/AML-71-001_AD.csv, read_counts/AML-71-001_DP.csv
    output: input/AML-71-001_input.csv, input/AML-71-001.loci
    jobid: 71
    wildcards: patient=AML-71-001

[Fri Jan 29 23:23:21 2021]
Error in rule discretize_counts:
    jobid: 99
    output: input/AML-99-001_input.csv, input/AML-99-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp88n8hmil.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:21 2021]
rule discretize_counts:
    input: read_counts/AML-53-001_AD.csv, read_counts/AML-53-001_DP.csv
    output: input/AML-53-001_input.csv, input/AML-53-001.loci
    jobid: 53
    wildcards: patient=AML-53-001

[Fri Jan 29 23:23:21 2021]
Error in rule discretize_counts:
    jobid: 94
    output: input/AML-94-001_input.csv, input/AML-94-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpq7wr92ob.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:21 2021]
rule discretize_counts:
    input: read_counts/AML-30-001_AD.csv, read_counts/AML-30-001_DP.csv
    output: input/AML-30-001_input.csv, input/AML-30-001.loci
    jobid: 32
    wildcards: patient=AML-30-001

[Fri Jan 29 23:23:21 2021]
Error in rule discretize_counts:
    jobid: 38
    output: input/AML-36-001_input.csv, input/AML-36-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpb3cscm6g.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:21 2021]
rule discretize_counts:
    input: read_counts/AML-18-001_AD.csv, read_counts/AML-18-001_DP.csv
    output: input/AML-18-001_input.csv, input/AML-18-001.loci
    jobid: 20
    wildcards: patient=AML-18-001

[Fri Jan 29 23:23:22 2021]
Error in rule discretize_counts:
    jobid: 58
    output: input/AML-58-001_input.csv, input/AML-58-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpp0o95_q0.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:22 2021]
rule discretize_counts:
    input: read_counts/AML-25-001_AD.csv, read_counts/AML-25-001_DP.csv
    output: input/AML-25-001_input.csv, input/AML-25-001.loci
    jobid: 27
    wildcards: patient=AML-25-001

[Fri Jan 29 23:23:23 2021]
Error in rule discretize_counts:
    jobid: 53
    output: input/AML-53-001_input.csv, input/AML-53-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7ho2k_fg.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:23 2021]
rule discretize_counts:
    input: read_counts/AML-68-001_AD.csv, read_counts/AML-68-001_DP.csv
    output: input/AML-68-001_input.csv, input/AML-68-001.loci
    jobid: 68
    wildcards: patient=AML-68-001

[Fri Jan 29 23:23:23 2021]
Error in rule discretize_counts:
    jobid: 71
    output: input/AML-71-001_input.csv, input/AML-71-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp8jro09eb.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:23 2021]
rule discretize_counts:
    input: read_counts/AML-33-001_AD.csv, read_counts/AML-33-001_DP.csv
    output: input/AML-33-001_input.csv, input/AML-33-001.loci
    jobid: 35
    wildcards: patient=AML-33-001

[Fri Jan 29 23:23:24 2021]
Error in rule discretize_counts:
    jobid: 20
    output: input/AML-18-001_input.csv, input/AML-18-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp2zao70xk.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:24 2021]
rule discretize_counts:
    input: read_counts/AML-42-001_AD.csv, read_counts/AML-42-001_DP.csv
    output: input/AML-42-001_input.csv, input/AML-42-001.loci
    jobid: 43
    wildcards: patient=AML-42-001

[Fri Jan 29 23:23:25 2021]
Error in rule discretize_counts:
    jobid: 32
    output: input/AML-30-001_input.csv, input/AML-30-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpyygervmb.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:25 2021]
rule discretize_counts:
    input: read_counts/AML-03-001_AD.csv, read_counts/AML-03-001_DP.csv
    output: input/AML-03-001_input.csv, input/AML-03-001.loci
    jobid: 3
    wildcards: patient=AML-03-001

[Fri Jan 29 23:23:25 2021]
Error in rule discretize_counts:
    jobid: 27
    output: input/AML-25-001_input.csv, input/AML-25-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp_f95eqik.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:25 2021]
rule discretize_counts:
    input: read_counts/AML-35-001_AD.csv, read_counts/AML-35-001_DP.csv
    output: input/AML-35-001_input.csv, input/AML-35-001.loci
    jobid: 37
    wildcards: patient=AML-35-001

[Fri Jan 29 23:23:26 2021]
Error in rule discretize_counts:
    jobid: 35
    output: input/AML-33-001_input.csv, input/AML-33-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpboiuwr6j.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:26 2021]
rule discretize_counts:
    input: read_counts/AML-54-001_AD.csv, read_counts/AML-54-001_DP.csv
    output: input/AML-54-001_input.csv, input/AML-54-001.loci
    jobid: 54
    wildcards: patient=AML-54-001

[Fri Jan 29 23:23:27 2021]
Error in rule discretize_counts:
    jobid: 68
    output: input/AML-68-001_input.csv, input/AML-68-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpqd9mt8ua.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:27 2021]
rule discretize_counts:
    input: read_counts/AML-12-001_AD.csv, read_counts/AML-12-001_DP.csv
    output: input/AML-12-001_input.csv, input/AML-12-001.loci
    jobid: 14
    wildcards: patient=AML-12-001

[Fri Jan 29 23:23:27 2021]
Error in rule discretize_counts:
    jobid: 3
    output: input/AML-03-001_input.csv, input/AML-03-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp66ad4lf2.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:27 2021]
rule discretize_counts:
    input: read_counts/AML-86-001_AD.csv, read_counts/AML-86-001_DP.csv
    output: input/AML-86-001_input.csv, input/AML-86-001.loci
    jobid: 86
    wildcards: patient=AML-86-001

[Fri Jan 29 23:23:28 2021]
Error in rule discretize_counts:
    jobid: 43
    output: input/AML-42-001_input.csv, input/AML-42-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmppo1p5htf.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:28 2021]
rule discretize_counts:
    input: read_counts/AML-64-001_AD.csv, read_counts/AML-64-001_DP.csv
    output: input/AML-64-001_input.csv, input/AML-64-001.loci
    jobid: 64
    wildcards: patient=AML-64-001

[Fri Jan 29 23:23:28 2021]
Error in rule discretize_counts:
    jobid: 37
    output: input/AML-35-001_input.csv, input/AML-35-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpkto7l630.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:29 2021]
rule discretize_counts:
    input: read_counts/AML-75-001_AD.csv, read_counts/AML-75-001_DP.csv
    output: input/AML-75-001_input.csv, input/AML-75-001.loci
    jobid: 75
    wildcards: patient=AML-75-001

[Fri Jan 29 23:23:29 2021]
Error in rule discretize_counts:
    jobid: 54
    output: input/AML-54-001_input.csv, input/AML-54-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpxkdtbonj.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:29 2021]
rule discretize_counts:
    input: read_counts/AML-46-001_AD.csv, read_counts/AML-46-001_DP.csv
    output: input/AML-46-001_input.csv, input/AML-46-001.loci
    jobid: 46
    wildcards: patient=AML-46-001

[Fri Jan 29 23:23:29 2021]
Error in rule discretize_counts:
    jobid: 14
    output: input/AML-12-001_input.csv, input/AML-12-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpqgmp5f1p.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:29 2021]
rule discretize_counts:
    input: read_counts/AML-91-001_AD.csv, read_counts/AML-91-001_DP.csv
    output: input/AML-91-001_input.csv, input/AML-91-001.loci
    jobid: 91
    wildcards: patient=AML-91-001

[Fri Jan 29 23:23:30 2021]
Error in rule discretize_counts:
    jobid: 86
    output: input/AML-86-001_input.csv, input/AML-86-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp71zgmebj.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:30 2021]
rule discretize_counts:
    input: read_counts/AML-61-001_AD.csv, read_counts/AML-61-001_DP.csv
    output: input/AML-61-001_input.csv, input/AML-61-001.loci
    jobid: 61
    wildcards: patient=AML-61-001

[Fri Jan 29 23:23:31 2021]
Error in rule discretize_counts:
    jobid: 64
    output: input/AML-64-001_input.csv, input/AML-64-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmphx0i4j9v.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:31 2021]
rule discretize_counts:
    input: read_counts/AML-47-001_AD.csv, read_counts/AML-47-001_DP.csv
    output: input/AML-47-001_input.csv, input/AML-47-001.loci
    jobid: 47
    wildcards: patient=AML-47-001

[Fri Jan 29 23:23:32 2021]
Error in rule discretize_counts:
    jobid: 75
    output: input/AML-75-001_input.csv, input/AML-75-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpq1ch8wtn.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:23:32 2021]
rule discretize_counts:
    input: read_counts/AML-07-001_AD.csv, read_counts/AML-07-001_DP.csv
    output: input/AML-07-001_input.csv, input/AML-07-001.loci
    jobid: 6
    wildcards: patient=AML-07-001

Terminating processes on user request, this might take some time.
[Fri Jan 29 23:23:32 2021]
Error in rule discretize_counts:
    jobid: 6
    output: input/AML-07-001_input.csv, input/AML-07-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpl6mj9vy0.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:23:32 2021]
Error in rule discretize_counts:
    jobid: 47
    output: input/AML-47-001_input.csv, input/AML-47-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpe_zh_v9y.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:23:32 2021]
Error in rule discretize_counts:
    jobid: 91
    output: input/AML-91-001_input.csv, input/AML-91-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpxje3sccz.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:23:32 2021]
Error in rule discretize_counts:
    jobid: 61
    output: input/AML-61-001_input.csv, input/AML-61-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpm8_q3lx4.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:23:32 2021]
Error in rule discretize_counts:
    jobid: 46
    output: input/AML-46-001_input.csv, input/AML-46-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpnel1opi9.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
Complete log: /scratch/data/leah/phyolin_private/aml_patients/.snakemake/log/2021-01-29T232243.219175.snakemake.log
